content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-certs.md

```diff
@@ -46,6 +46,7 @@ For more details see [Manual certificate renewal](/docs/tasks/administer-cluster
 {{< tab name="etcd-server" include="generated/kubeadm_certs_renew_etcd-server.md" />}}
 {{< tab name="front-proxy-client" include="generated/kubeadm_certs_renew_front-proxy-client.md" />}}
 {{< tab name="scheduler.conf" include="generated/kubeadm_certs_renew_scheduler.conf.md" />}}
+{{< tab name="super-admin.conf" include="generated/kubeadm_init_phase_kubeconfig_super-admin.conf.md" />}}
```

Comment:
This file is required to enable the ci success.

Reply:
That was a typo.